plugins/lookup/conjur_variable.py

```diff
@@ -203,12 +210,24 @@ def _fetch_conjur_variable(conjur_variable, token, conjur_url, account, validate
     return {}
 
 
+def _store_in_file(value):
+    file_name = ''.join(choice(string.ascii_uppercase + string.digits) for i in range(12))
+    file_path = os.path.join("/dev/shm", file_name)
+    with open(file_path, 'w+') as file:
+        file.write(value[0])
+        os.chmod(file_path, stat.S_IRUSR | stat.S_IWUSR)
```

Comment:
We should chmod the file before we write the secret into it

Reply:
Fixed in next commit